tests/integration/components/frost-bunsen-detail/conditional-views-test.js

```diff
@@ -0,0 +1,81 @@
+import {expect} from 'chai'
+import {setupComponentTest} from 'ember-mocha'
+import wait from 'ember-test-helpers/wait'
+import hbs from 'htmlbars-inline-precompile'
+import {describe, it, beforeEach} from 'mocha'
+
+function renderWithValue (context, value) {
+  context.set('value', value)
+  context.render(hbs`{{frost-bunsen-form
+    bunsenModel=bunsenModel
+    bunsenView=bunsenView
+    hook=hook
+    value=value
+  }}`)
+}
+
+describe('Integration: frost-bunsen-form / views with conditions', function () {
+  setupComponentTest('forms with conditions', {
```

Comment:
Mainly did it this way because I wanted to control when/where the call to `this.render` was being made. I had written it using the helpers but I rewrote that part when debugging an issue.